tests/Doctrine/Tests/ORM/Functional/Ticket/DDC7079Test.php

```diff
@@ -0,0 +1,126 @@
+<?php
+declare(strict_types=1);
+
+namespace Doctrine\Tests\ORM\Functional\Ticket;
+
+use Doctrine\Common\Persistence\Mapping\RuntimeReflectionService;
+use Doctrine\ORM\Mapping\ClassMetadata;
+use Doctrine\ORM\Mapping\DefaultQuoteStrategy;
+use Doctrine\Tests\OrmFunctionalTestCase;
+
+/**
+ * Class DDC7079Test
+ * @group DDC-7079
+ * @package Doctrine\Tests\ORM\Functional
+ * @author mairo744
+ */
+class DDC7079Test extends OrmFunctionalTestCase
+{
+    /**
+     * @var \Doctrine\ORM\Mapping\DefaultQuoteStrategy
+     */
+    private $strategy;
+
+    /**
+     * @var \Doctrine\DBAL\Platforms\AbstractPlatform
+     */
+    private $platform;
+
+    /**
+     * {@inheritDoc}
+     */
+    protected function setUp()
+    {
+        parent::setUp();
+        $this->platform = $this->_em->getConnection()->getDatabasePlatform();
+        $this->strategy = new DefaultQuoteStrategy();
+    }
+
+    public function testGetTableName ()
+    {
+        $table = [
+            'name'=>'cms_user',
```

Comment:
This does not look like it should be ok with our coding standard… maybe it changed since February? Can you please rebase, and take that as an occasion to squash all your commits into one commit?

Making sure you follow [this paragraph of the contributing guide](https://www.doctrine-project.org/contribute/index.html#working-on-topic-branches) when you do.

Reply:
As predicted, the build fails because of cs issues: https://travis-ci.org/doctrine/orm/jobs/607884252?utm_medium=notification&utm_source=github_status

Running `vendor/bin/phpcbf` should fix them.

Reply:
@mairo744 I've just created a PR to your branch that fixes cs issues

Reply:
Thank you @greg0ire and @ajgarlag for help :)

Reply:
@ajgarlag it's quite rare to see people use git in a decentralized way. I love it when people really start to collaborate on things like this. Great job moving this forward!